### PR TITLE
GRT: fix the upload logging init

### PR DESCRIPTION
**Pull request by martenson**
- properly initialiaze the logger
- change level for some of the logs
- add some debug and critical logs

### CODE REVIEW

**Comment by martenson** [comment]
added a commit to expand default sanitation, I think data source tools are safe to ignore

File: scripts/grt/grt.yml.sample [hunk 1]
```diff
@@ -21,6 +21,23 @@ sanitization:
     tools:
         - __SET_METADATA__
         - upload1
+        - ucsc_table_direct1
```
Comment (bgruening) [inline scripts/grt/grt.yml.sample]: @martenson I find these information super useful, any reason to blacklist those?
https://stats.usegalaxy.eu/d/SDduH5Zik/grt-tool-usage?refresh=15m&orgId=1&var-tool_id=ucsc_table_direct1&var-instance_id=All&var-date_interval=30d

File: scripts/grt/upload.py [hunk 2]
```diff
@@ -24,14 +24,16 @@ def main(argv):
                         default=default_config)
     parser.add_argument("-l", "--loglevel", choices=['debug', 'info', 'warning', 'error', 'critical'],
                         help="Set the logging level", default='warning')
+
     args = parser.parse_args()
+    logging.getLogger().setLevel(getattr(logging, args.loglevel.upper()))
```
Comment (hexylena) [inline scripts/grt/upload.py:29]: Oh. Duh. Thanks.

**Review by hexylena — APPROVED** [review]
Could you also make the default for max-records in [export.py](https://github.com/martenson/galaxy/blob/390aedf8c8c28db739a78dc7c41e9275a735369b/scripts/grt/export.py#L138) something like 5M?

File: scripts/grt/grt.yml.sample (hunk 1 above)
Comment (martenson) [reply scripts/grt/grt.yml.sample]: Hmm, I guess the execution numbers could be useful. I was more thinking in terms of parameters, runtime, metrics etc. which are nonexistent/useless. I will revert this.

**Comment by martenson** [comment]
@erasche set to 5M
@bgruening reverted the sanitation

thanks for review!

**Pull request merged**